src/app/modules/main/profile_section/wallet/networks/module.nim

```diff
@@ -98,6 +98,7 @@ method toggleTestNetworksEnabled*(self: Module) =
 
 method updateNetworkEndPointValues*(self: Module, chainId: int, newMainRpcInput, newFailoverRpcUrl: string) =
   self.controller.updateNetworkEndPointValues(chainId, newMainRpcInput, newFailoverRpcUrl)
+  self.refreshNetworks()
```

Comment:
@Khushboo-dev-cpp you're refreshing networks here regardless to the result of the action in previous step. Would be much better if you would connect to signal `SIGNAL_NETWORK_ENDPOINT_UPDATED` emitted from the previous action and refresh network only in that case.